src/components/NavBar/NavLinks.js

```diff
@@ -0,0 +1,61 @@
+import i18next from "i18next";
+import { useTranslation, initReactI18next } from "react-i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+
+const lngs = {
+  en: { nativeName: "English" },
+  pl: { nativeName: "Polski" },
+};
+
+i18next
+  .use(initReactI18next)
+  .use(LanguageDetector)
+  .init({
+    debug: true,
+    fallbackLng: "en",
+    lng: "en",
+    resources: {
+      en: {
+        translation: {
+          invoices: "INVOICES",
+        },
+      },
+      pl: {
+        translation: {
+          invoices: "FAKTURY",
+        },
+      },
+    },
+  });
+
+const NavLinks = () => {
+  const { t, i18n } = useTranslation();
+
+  return (
+    <ul>
+      <li>
+        <a href="#">{t("invoices")}</a>
```

Comment:
import {Link}from "react-router-dom";

![image](https://user-images.githubusercontent.com/64959616/226624258-a8367b01-5202-46e9-a24a-6e63e6ce31ec.png)

Reply:
O, nie wiedziałam, że react ma od tego moduł Link. Dzięki!

Reply:
O, nie wiedziałam, że react ma od tego moduł Link. Dzięki!

Reply:
to jest element z react-router-dom :)